packages/web3-shared/base/src/utils/currency.ts

```diff
@@ -63,3 +63,7 @@ export function formatCurrency(value: BigNumber.Value, currency = 'USD'): string
         })
         .reduce((string, part) => string + part)
 }
+
+export const ethFormatter = (value: BigNumber.Value, sign = '\u039E') => {
+    return `${new BigNumber(value).toFixed(2)} ${sign}`
```

Comment:
I think `fromatCurrency` is the prior choice for formatting currency.

Reply:
It's different from `fromatCurrency` that the symbol is after digits.